test/Microsoft.TemplateEngine.Mocks/MockFileSystem.cs

```diff
@@ -237,5 +237,15 @@ public DateTime GetLastWriteTimeUtc(string file)
 
             return _files[file].LastWriteTimeUtc;
         }
+
+        public void SetLastWriteTimeUtc(string file, DateTime lastWriteTimeUtc)
+        {
+            if (!FileExists(file))
+            {
+                throw new Exception($"File {file} doesn't exist");
+            }
+
+            _files[file].LastWriteTimeUtc = lastWriteTimeUtc;
```

Comment:
minor: is it possible `_files` would not contain file?  If yes, are we ok throwing here?  Probably yes, as we are throwing an exception above.

Reply:
Yes, this just preserves the behavior of all of the other APIs.